sct.py

```diff
@@ -1456,7 +1456,8 @@ def create_test_release_jobs_enterprise(branch, username, password, sct_branch,
         ('ICS', '*ics*', 'ICS'),
         ('Workload_Prioritization', 'features-sla-*', 'Workload Prioritization'),
         ('LDAP', '*ldap*', 'LDAP authorization and authentication tests'),
-        ('LDAP', '*saslauthd*', 'LDAP authorization and authentication tests')
+        ('LDAP', '*saslauthd*', 'LDAP authorization and authentication tests'),
+        ('FIPS', '*fips*', 'FIPS Encryption tests')
```

Comment:
please a comma for the next time we need to add something
```suggestion
        ('FIPS', '*fips*', 'FIPS Encryption tests'),
```

Reply:
Done